content/blog/the-world-of-web-rag/index.md

```diff
@@ -0,0 +1,359 @@
+---
+layout: blog-post
+title: The World of Web RAG
+description: Let's discover how to make your LLMs make a quick Google search before answering your questions
+featured_image: thumbnail.jpg
+images: ["blog/the-world-of-web-rag/thumbnail.jpg"]
+alt_image: A person water-skiing in front of the Rose Island sea platform in a vintage picture.
+toc: True
+date: 2023-11-09
+last_updated:  2023-11-09
+authors:
+  - Sara Zanzottera
+tags: ["Haystack 2.0"]
+---	
+
+In an earlier post of the Haystack 2.0 series, we've seen how to build RAG and indexing pipelines. An application that uses these two pipelines is practical if you have an extensive, private collection of documents and need to perform RAG on such data only. However, in many cases, you may want to get data from the Internet: from news outlets, documentation pages, and so on.
+
+In this post, we will see how to build a Web RAG application: a RAG pipeline that can search the Web for the information needed to answer your questions.
+
+> 💡 *Do you want to see the code in action? Check out the [Colab notebook](https://colab.research.google.com/drive/1dGMPxReo730j7_zQDZOu-0SGf-pk4XDL?usp=sharing) or the [gist](https://gist.github.com/ZanSara/0907a8f3ae19f62998cc061ed6e8ce53).*
+
+> <i>⚠️ **Warning:**</i> *This code was tested on `haystack-ai==0.130.0`. Haystack 2.0 is still unstable, so later versions might introduce breaking changes without notice until Haystack 2.0 is officially released. The concepts and components, however, stay the same.*
+
+
+# Searching the Web
+
+As we've seen [earlier](#), a Haystack RAG Pipeline is made of three components: a Retriever, a PromptBuilder, and a Generator, and looks like this:
```

Comment:
This was linking to a post that we haven't published here yet. LMK what to point it to.

Reply:
just remove the first bit omi, Start with 'A Haystack RAG pipeline...'